src/robot_properties_solo/solo12wrapper.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         if orn is None:
             orn = pybullet.getQuaternionFromEuler([0, 0, 0])
 
-        pybullet.setAdditionalSearchPath(Solo12Config.meshes_path)
+        pybullet.setAdditionalSearchPath(Solo12Config.resources.package_path)
```

Comment:
it is good as is. but you could define a mesh path for the pybullet if needed.

Reply:
got it